### PR TITLE
Cwf helmchart changes

### PR DESCRIPTION
**Pull request by reddydodda**
This commit fix below issues

1. split docker dpi compose file to have a separate build file

2. using override variable for interfaces mapping for cwf

3. add a quote for nms db secrets

4. bump helm chart version

### CODE REVIEW

**Comment by reddydodda** [comment]
@mpgermano  @koolzz  please review this PR.

File: orc8r/tools/docker/install_gateway.sh [hunk 1]
```diff
@@ -89,7 +89,7 @@ if [ "$GW_TYPE" == "$CWAG" ] || [ "$GW_TYPE" == "$XWF" ]; then
   apt-add-repository -y ppa:ansible/ansible
   apt-get update -y
   apt-get -y install ansible
-  ANSIBLE_CONFIG="$INSTALL_DIR"/magma/"$MODULE_DIR"/gateway/ansible.cfg ansible-playbook "$INSTALL_DIR"/magma/"$MODULE_DIR"/gateway/deploy/cwag.yml -i "localhost," -c local -v -e ingress_port="eth1" -e uplink_ports="eth2 eth3" -e li_port="eth4"
+  ANSIBLE_CONFIG="$INSTALL_DIR"/magma/"$MODULE_DIR"/gateway/ansible.cfg ansible-playbook "$INSTALL_DIR"/magma/"$MODULE_DIR"/gateway/deploy/cwag.yml -i "localhost," -c local -v -e ingress_port="${INGRESS_PORT:-eth1}" -e uplink_ports="${UPLINK_PORTS:-eth2 eth3}" -e li_port="${LI_PORT:-eth4}"
```
Comment (mpgermano) [inline orc8r/tools/docker/install_gateway.sh:92]: Will this cause issues if the env vars INGRESS_PORT, UPLINK_PORTS, LI_PORT aren't defined? It's likely the xwf gateway .env won't have these variables

Comment (reddydodda) [reply orc8r/tools/docker/install_gateway.sh:92]: @mpgermano  if INGRESS_PORT is not defined then default is eth1.  same for others.

**Review by mpgermano — APPROVED** [review]

**Comment by 119Vik** [comment]
Can one of the admins verify this patch?

**Pull request closed without merging**